specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/network.json

```diff
@@ -0,0 +1,439 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2022-06-01-preview"
```

Comment:
>2022-06-01-preview [](http://example.com/codeflow?start=16&length=18)

Why is this a backdated API?

Reply:
blocking the PR to get an answer to this question. The rest of the changes are fine